src/help/help_impl.cpp

```diff
@@ -340,35 +343,56 @@ const std::vector<std::string>& topic_text::parsed_text() const
 	return parsed_text_;
 }
 
+std::string time_of_day_bonus_colored(const int time_of_day_bonus)
+{
+	// Use same red/green colouring scheme as time_of_day_at() in reports.cpp for consistency
+	return std::string("<format>color='") + (time_of_day_bonus > 0 ? "green" : (time_of_day_bonus < 0 ? "red" : "white")) + "' text='" + std::to_string(time_of_day_bonus) + "'</format>";
+}
+
 std::vector<topic> generate_time_of_day_topics(const bool /*sort_generated*/)
 {
 	std::vector<topic> topics;
 	std::stringstream toplevel;
 
 	if (! resources::tod_manager) {
 		toplevel << _("Only available during a scenario.");
-		topics.emplace_back("Time of Day Schedule", "..schedule", toplevel.str());
+		topics.emplace_back(_("Time of Day Schedule"), "..schedule", toplevel.str());
 		return topics;
 	}
+
 	const std::vector<time_of_day>& times = resources::tod_manager->times();
 	for (const time_of_day& time : times)
 	{
 		const std::string id = "time_of_day_" + time.id;
 		const std::string image = "<img>src='" + time.image + "'</img>";
+		const std::string image_lawful = "<img>src='icons/alignments/alignment_lawful_30.png'</img>";
+		const std::string image_neutral = "<img>src='icons/alignments/alignment_neutral_30.png'</img>";
+		const std::string image_chaotic = "<img>src='icons/alignments/alignment_chaotic_30.png'</img>";
+		const std::string image_liminal = "<img>src='icons/alignments/alignment_liminal_30.png'</img>";
 		std::stringstream text;
 
-		toplevel << make_link(time.name.str(), id) << jump_to(160) <<
-				image << jump(30) << time.lawful_bonus << '\n';
+		const int lawful_bonus = generic_combat_modifier(time.lawful_bonus, unit_type::ALIGNMENT::LAWFUL, false, resources::tod_manager->get_max_liminal_bonus());
+		const int neutral_bonus = generic_combat_modifier(time.lawful_bonus, unit_type::ALIGNMENT::NEUTRAL, false, resources::tod_manager->get_max_liminal_bonus());
+		const int chaotic_bonus = generic_combat_modifier(time.lawful_bonus, unit_type::ALIGNMENT::CHAOTIC, false, resources::tod_manager->get_max_liminal_bonus());
+		const int liminal_bonus = generic_combat_modifier(time.lawful_bonus, unit_type::ALIGNMENT::LIMINAL, false, resources::tod_manager->get_max_liminal_bonus());
+
+		toplevel << make_link(time.name.str(), id) << jump_to(160) << image << jump(30) <<
+			image_lawful << time_of_day_bonus_colored(lawful_bonus) << " / " <<
+			image_neutral << time_of_day_bonus_colored(neutral_bonus) << " / " <<
+			image_chaotic << time_of_day_bonus_colored(chaotic_bonus) << " / " <<
+			image_liminal << time_of_day_bonus_colored(liminal_bonus) << '\n';
```

Comment:
Screenshot for convenience:

![2019-10-24-063802_569x310_scrot](https://user-images.githubusercontent.com/24784687/67459788-df7ac300-f628-11e9-910c-aecaa9bd8b53.png)

Looks good! I wonder if it's easy to make all the "chaotic" icons vertically aligned with each other? There's a `jump` tag, I think that's what the tables in unit type topics use?

(I mean, the chaotic icon is [not actually a possible moon phase](https://xkcd.com/1738/), but that's not something this PR needs to solve)

Reply:
I did consider the `jump` function but by the time I got to this point it was a long work day and didn't think it worth the effort, but I suppose the time-of-day images are fixed width, so I'll try to revisit this.

Reply:
Another problem: This format won't work for 800x600, but I think 1024x768 is sufficient.

Reply:
1024x768 is insufficient with a long enough translation or time-of-day string, but that's #1167 again.

I managed to make something that should work well on high enough resolutions.

Forgot to mention that I've seen that comic as well. I'm aware of the phases of the moon and that a crescent moon is more commonly used to indicate night. I can only guess the artist was trying to have something that could combine with lawful to make the neutral symbol. That, or he/she was being especially careful to avoid political-correctness backlash.

Reply:
Looks good!